docs-chef-io/content/inspec/resources/azure_active_directory_domain_service.md

```diff
@@ -77,15 +77,15 @@ Either one of the following parameters is mandatory.
 
 ## Examples
 
-**Test If an Active Directory Domain is Referenced with a Valid ID.**
+### Test if an active directory domain is referenced with a valid ID
```

Comment:
These headings should stay bolded.

Reply:
Hey Ian -- we discussed them to remove as the heading tag itself meets the requirement and also it looks odd to have a complete sentence bolded. This way I have written guidelines and reviewed all Azure resources and almost AWS and InSpec resources